src/formula_graph.rs

```diff
@@ -402,7 +402,7 @@ impl<'a> DataFlowGraphBuilder<'a> {
                 let to_add = 8 - (size % 8);
                 let words_read = (size + to_add) / 8;
 
-                for i in 0..words_read {
+                for i in 0..words_read - 1 {
```

Comment:
I think that change is wrong. Say `size = 7`, then `words_read = 1`, so the range is `[0, 1[` which is `0..1` in Rust. So one (double) word get's read. Whereas with your change, for the range `0..0`, nothing would be read.